doc/appendix-de-kassensichv/reference-tables/reference-tables.md

```diff
@@ -279,3 +279,474 @@ For definitions regarding national laws, please refer to the appropriate appendi
+| `TERMINAL_ID` | ID of the terminal | String (50) |Mandatory, `dailyClosingMasterData.Terminals.Id` |
+| `TERMINAL_BRAND` | Brand of the terminal | String (50) | Mandatory, `dailyClosingMasterData.Terminals.Brand`|
+| `TERMINAL_MODELL` | Model designation | String (50) | Mandatory, `dailyClosingMasterData.Terminals.Model` |
+| `TERMINAL_SERIENNR` | Serial number of the terminal | String (70) | Mandatory, `dailyClosingMasterData.Terminals.SerialNr` |
+| `TERMINAL_SW_BRAND` | Brand name of the software | String (50) | Mandatory, `dailyClosingMasterData.Terminals.SWBrand` |
+| `TERMINAL_SW_VERSION` | Version of the software | String (50) | Mandatory, `dailyClosingMasterData.Terminals.SWVersion` |
+
+(`dailyClosingMasterData`is described above in the chapter "Master data module")
+
+##### File: Stamm_Agenturen (pa.csv)
+
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | From `cbReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically filled by ft |
+| `AGENTUR_ID` | ID of the agency | Integer | Mandatory, `dailyClosingMasterData.Agencies.Id` | 
+| `AGENTUR_NAME` | Name of the client | String (60) | Mandatory, `dailyClosingMasterData.Agencies.Name` |
+| `AGENTUR_STRASSE` | Street | String (60) | Mandatory, `dailyClosingMasterData.Agencies.Street`|
+| `AGENTUR_PLZ` | Zip | String (10) |  Mandatory, `dailyClosingMasterData.Agencies.Zip` |
+| `AGENTUR_ORT` | City | String (62) | Mandatory, `dailyClosingMasterData.Agencies.City` |
+| `AGENTUR_LAND` | ISO 3166 ALPHA-3 country code | String | Mandatory, `dailyClosingMasterData.Agencies.Country` |
+| `AGENTUR_STNR` | Tax number | String (20) | Mandatory, `dailyClosingMasterData.Agencies.TaxNr` |
+| `AGENTUR_USTID` | VAT ID| String (15) | Mandatory, `dailyClosingMasterData.Agencies.VATId` |
+
+(`dailyClosingMasterData`is described above in the chapter "Master data module")
+
+##### File: Stamm_USt (vat.csv)
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | From `cbReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically filled by ft  |
+| `UST_SCHLUESSEL` | ID of the VAT rate| Integer | Automatically filled by ft  |
+| `UST_SATZ` | Percentage | Decimal (2) | Automatically filled by ft  |
+| `UST_BESCHR` | Description| String | Automatically filled by ft  |
```

Comment:
The calculation by the middleware must be adapted.

Reply:
HI @AxelKutschera , thank you for this comment. Can you please explain it more detailed?

Reply:
Till now, the Middleware didn´t calculat the vat. like: 
- POS_BRUTTO / (1+"VAT% depending on `ftChargeItemCase") = POS_NETTO
- POS_NETTO * "VAT% depending on `ftChargeItemCase" = POS_UST
- how to round?

e.g. also how ft shall provide the calcultaion response in lines 382-384 and also in lines 432-434

Reply:
True, thank you for the hint. We need to think how to round. And I think that the calculation is only needed in the process of exporting the dsfinv-k. Therefore the description would be an implementation detail but this chapter refers on how the cashpoint can send the needed data to the ft.Middleware and if not needed, where the data comes from.

Reply:
this will be part of the implementation specification